src/libs/hashCode.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-bitwise */
```

Comment:
I'm guessing that we don't like bitwise operations because of the linter complaining, but in this case this is a very atomic copy pasted function that is making some mathematical operations. I don't care about the details of the operation and I know that it could be replaced by a multiplication or other, but from the comments on the source, it appeared that it performed better with the bitwise operation.

Reply:
I'm fine with using bitwise operators here for a function copy/pasted from SO